curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6554a88d5af937226f4a9121.md

```diff
@@ -1,8 +1,8 @@
 ---
 id: 6554a88d5af937226f4a9121
-title: Step 60
+title: Step 64
 challengeType: 20
-dashedName: step-60
+dashedName: step-64
 ---
 
 # --description--
```

Comment:
Modified instruction:
After incrementing the `key_index`, create a variable named `offset`. Find the position of `key_char` in the `alphabet` and set this position as the value of `offset`. Use the `.index()` method to find the position.